cmd/qgb/orchestrator/config.go

```diff
@@ -102,7 +106,11 @@ func parseOrchestratorFlags(cmd *cobra.Command) (StartConfig, error) {
 }
 
 func addInitFlags(cmd *cobra.Command) *cobra.Command {
-	cmd.Flags().String(base.FlagHome, "", "The qgb orchestrator home directory")
+	homeDir, err := base.DefaultServicePath("orchestrator")
```

Comment:
[not blocking]
we're using "orchestrator" twice here, so we might want to consider using a constant.

Reply:
https://github.com/celestiaorg/orchestrator-relayer/issues/320